integration/tests/basic/rpc/test_rpc_get_block.py

```diff
@@ -99,9 +110,20 @@ def test_eth_get_block_by_number_with_incorrect_data(self, json_rpc_client):
     @pytest.mark.bug  # fails on geth (returns a different error message), needs a fix, and refactor of Error32602
     def test_eth_get_block_by_number_with_not_exist_data(self, number, full_trx, json_rpc_client):
         """Verify implemented rpc calls work eth_getBlockByNumber"""
-        response = json_rpc_client.send_rpc(method="eth_getBlockByNumber", params=[gen_hash_of_block(number), full_trx])
+        response = json_rpc_client.send_rpc(method="eth_getBlockByNumber", params=[gen_hash_of_block(number), full_trx]) 
+        EthGetBlockByHashResult(**response)
         assert "result" in response and response["result"] is None, "Result should be None"
+    
+    @pytest.mark.xfail(reason="NDEV-3072")  # fails on geth (returns a different error message), needs a fix, and refactor of Error32602
+    @pytest.mark.parametrize("full_trx", [False, True])
+    def test_eth_get_block_by_number_with_big_int(self, full_trx, json_rpc_client):
+        """Verify implemented rpc calls work eth_getBlockByNumber"""
+        response = json_rpc_client.send_rpc(method="eth_getBlockByNumber", params=["0x55192d7d9e36433b64f2b9d9309a5c5d36b1561c888dcfa8f31078f000fa7cdd", full_trx])
+        EthError32602(**response)
 
+    # geth returns miner as null, totalDifficulty as null, for
+    # Tag.LATEST-True, Tag.PENDING-True, Tag.PENDING-False
+    @pytest.mark.xfail(reason="NDEV-3170") # verify after DOPS-723 is done
```

Comment:
if schema is different now let's not mark with xfail all tests now
Because there are other important checks and we can miss some new bugs as I don't how fast schema will be fixed

Reply:
fixed ✅